src/content/docs/browser/new-relic-browser/configuration/experimental-features.mdx

```diff
@@ -14,12 +14,14 @@ The following experimental features are available in New Relic Browser:
 
 
 <Callout variant="important">
-  Experimental features are available only for opt-in on manual copy and paste, or NPM implementations of the agent. These features are subject to change and should be used with caution.
+  Experimental features are available only for opt-in manually with copy and paste or NPM implementations of the agent. To gain access for APM-injected applications, reach out to your support representative. Experimental features are subject to change and should be used with caution.
 </Callout>
 
-## Opt in to use experimental features
+## Manually opting-in to use experimental features
 
-### Browser Performance - Marks, Measures & Resources
+### Copy/Paste Implementations
+
+#### Browser Performance - Marks, Measures & Resources
```

Comment:
Here, "Copy/Paste Implementations" is a heading and under this, "Browser Performance - Marks, Measures & Resources" is another heading without any content. Also, the entire page is about Browser Performance. So, we can omit from the heading. So, my suggestion is to convert it to one heading as following:

###Manual  Implementation of Marks, Measures, or Page resources

Reply:
There are 3 installation patterns of the browser agent -- APM, NPM, and "Copy/Paste".  Copy/Paste here simply refers to the installation method, not anything to do with "manual implementation".  If you want to rephrase things, that is fine, but we need to ensure we are showing "how" to enable experimental features for each installation type.

Reply:
`BrowserPerformance` (btw) is the event type. `SELECT * FROM BrowserPerformance` for example